packages/expo-clipboard/src/Clipboard.types.ts

```diff
@@ -38,3 +38,32 @@ export interface ClipboardImage {
     height: number;
   };
 }
+
+export enum StringFormat {
```

Comment:
Let's add some comment here:
```suggestion
/**
 * Type used to determine string format stored in the clipboard.
 */
export enum StringFormat {
```